### PR TITLE
use realtime data for top picks page

### PR DESCRIPTION
**Pull request by radiodario**
removed mquery, use a single query to get the data from realtime, however, retention rate cannot be calculated, so we get this:
Imgur

what do I do about this? cc: @clairepowell

### CODE REVIEW

**Comment by jonathan-best** [comment]
Yep looks good to me - Merge away once Circle is happy

:+1:

File: src/shared/handlers/TopArticlesView.js [hunk 1]
```diff
@@ -32,31 +27,44 @@ function convertTime (avg) {
...
-  return data.map((d, i) => {
+  return data.map((d) => {
     let uuid = d.key;
     let metricVal = d[metric];
     let title =  d.title.buckets[0] ? d.title.buckets[0].key : "Unknown"
     let author = getAuthors(d);
     let articleUrl = (
       <a href={`/landing/article/${uuid}`}
-         target="_blank"
+        target="_blank"
```
Comment (DanForys) [inline src/shared/handlers/TopArticlesView.js]: :neutral_face:

**Comment by DanForys** [comment]
:es:

**Comment by jonathan-best** [comment]
Merged for Mr Villanueva :sunglasses:

**Pull request merged**